src/sage/schemes/toric/weierstrass.py

```diff
@@ -176,8 +176,8 @@ def Discriminant(polynomial, variables=None):
         sage: Discriminant([quadratic1, quadratic2])
         -1/16
     """
-    (f, g) = WeierstrassForm(polynomial, variables)
-    return 4*f**3+27*g**2
+    f, g = WeierstrassForm(polynomial, variables)
+    return 4 * f**3 + 27 * g**2
```

Comment:
```suggestion
    return 4*f**3 + 27*g**2
```
whitespace around top-level arith operators only please